.github/workflows/e2e.yaml

```diff
@@ -39,19 +39,21 @@ jobs:
           ./mvnw --batch-mode -Dcheckstyle.skip -Drat.skip -T2 -Dmaven.compile.fork -Dmaven.compiler.maxmem=3072 -DskipTests clean install
           ./mvnw --batch-mode -f test/e2e/pom.xml -pl e2e-base clean install
       - name: Single Node Tests(JDK8)
-        run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-single-service
+        run: export E2E_VERSION=jdk8-1.4 && bash -x test/e2e/run.sh e2e-single-service
+      - name: Single Node Tests(InfluxDB/JDK8)
+        run: export E2E_VERSION=jdk8-1.4 && bash -x test/e2e/run.sh e2e-influxdb
       - name: Single Node Tests(MySQL/JDK8)
-        run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-mysql
+        run: export E2E_VERSION=jdk8-1.4 && bash -x test/e2e/run.sh e2e-mysql
       - name: Single Node Tests(JDK9)
-        run: export E2E_VERSION=jdk9-1.3 && bash -x test/e2e/run.sh e2e-single-service
+        run: export E2E_VERSION=jdk9-1.4 && bash -x test/e2e/run.sh e2e-single-service
       - name: Single Node Tests(JDK11)
-        run: export E2E_VERSION=jdk11-1.3 && bash -x test/e2e/run.sh e2e-single-service
+        run: export E2E_VERSION=jdk11-1.4 && bash -x test/e2e/run.sh e2e-single-service
       - name: Single Node Tests(JDK12)
-        run: export E2E_VERSION=jdk12-1.3 && bash -x test/e2e/run.sh e2e-single-service
+        run: export E2E_VERSION=jdk12-1.4 && bash -x test/e2e/run.sh e2e-single-service
       - name: Agent Reboot Tests(JDK8)
-        run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-agent-reboot
+        run: export E2E_VERSION=jdk8-1.4 && bash -x test/e2e/run.sh e2e-agent-reboot
 
-  Cluster:
+  Cluster_N_TTL:
```

Comment:
What do you mean N_TTL?

Reply:
> What do you mean N_TTL?

Possibly `N` == `And`, we don't need to save 2 letters to cause confusion, no?

Reply:
Three things
1. N represent AND is not a common case. `+` and `&` are better
1. This name should not be changed randomly, `Cluster` is a required test, without this check(named as `cluster`), we can't merge PR unless we request the INFRA change the settings.
1. If you want the test group names more clear, moving the TTL out of the cluster test group makes sense to me. But notice, it will take more time as it requires to compile the project again.

Reply:
Yes. I will revert it.